### ISSUE TITLE
Can't paste password into Login/Create account prompt

### ISSUE THREAD

**Issue by html5cat** [exponentjs/xde#15]
XDE v0.2.2 on latest osx 10.11

**Comment by ide** [comment]
Thanks for the bug report! We'll look into it soon.

**Comment by ide** [comment]
Apparently this is a bug with the packaged version of the Mac app. If you clone this Git repo, run `npm install`, and then `npm start` it will make copy/paste work but you shouldn't have to do that.

**Comment by ccheever** [comment]
Hi @html5cat Thanks for the report.

You can use @ide's workaround or write some code that adds the appropriate menu items and keyboard shortcuts to the electron app and submit a patch. I'll plan on doing that shortly but feel free to make the fix yourself if you like.

**Comment by html5cat** [comment]
Thanks! I'll look into it

**Comment by html5cat** [comment]
I'm getting `Developer ID Application: 650 Industries, Inc. (C8D8QTF339): no identity found` – can you send me cert or can you point to where to update/create it? Thanks!

**Comment by ccheever** [comment]
Oh no! Sorry. I'm away from my computer but will fix It so you don't need
the certificate as soon as I'm back.

**Comment by ccheever** [comment]
Hi @html5cat - 

If you ran into the problem when running
`npm run-script package`
then that makes sense.

I don't want to release the cert publicly because then anyone could make forged releases of xde and I think its somewhat useful to be able to say a version came from us.

What I did was, I just pushed https://github.com/exponentjs/xde/commit/18d954346066e21d9e8a01ad2f1a7cebe1ce80a3 that makes `npm run-script package` *not* sign the packaged app.

(Now, to do the signed version, you do `npm run-script signed-package`, but you shouldn't need to bother with that)

Let me know if that helps!
